lib/videoroom_web/templates/layout/app.html.heex

```diff
@@ -18,7 +18,7 @@
         <%= @inner_content %>
       </main>
       <div>
-        <label class="absolute text-brand-dark-blue-500 text-xl bottom-5 left-3 invisible md:visible">Version: <%= version() %></label>
+        <label id="videoroom-version" class="absolute text-brand-dark-blue-500 text-xl bottom-5 left-3 invisible md:visible">Version: <%= version() %></label>
```

Comment:
This solution with conditionally hiding the label is really hacky. Instead, please move the div with the label to `templates/page/index.html.heex` - this way it won't be rendered on the stats page and seems to look fine anyway.

To achieve that you have to put  `Application.fetch_env!(:membrane_videoroom_demo, :version)` to params when rendering layout and use it inside the template

```elixir
# router.ex
    get("/healthcheck", PageController, :healthcheck)
    get("/webrtc-internals", PageController, :internals)

# page_controller.ex
defmodule VideoRoomWeb.PageController do
  use VideoRoomWeb, :controller

  @spec index(conn :: Plug.Conn.t(), params :: map()) :: Plug.Conn.t()
  def index(conn, params) do
    render(conn, "index.html",
      room_id: Map.get(params, "room_id"),
      simulcast: Map.get(params, "simulcast"),
      version: Application.fetch_env!(:membrane_videoroom_demo, :version)
    )
  end

  @spec internals(conn :: Plug.Conn.t(), params :: map()) :: Plug.Conn.t()
  def internals(conn, _params) do
    render(conn, "index.html")
  end
# ...

# page/index.html.heex

<div id="root"></div>

<%!-- TODO: Use :if in Phoenix 1.7 --%>
<%= if assigns[:version] do %>
<label class="absolute text-brand-dark-blue-500 text-xl bottom-5 left-3 invisible md:visible">Version: <%= @version %></label>
<% end %>

```